src/_data/survey-vers.json

```diff
@@ -0,0 +1,7 @@
+{
+  "uniqueId": "2020Q1",
+  "startMsEpoch": "2020/03/03 9:00",
+  "endMsEpoch": "2020/03/10 9:00",
+  "title": "Help improve Flutter! Take our Q1 survey.",
```

Comment:
I'd add comments to the file for this field, to indicate that the title has to be super brief; we're limited in IDEs as to how much we can display. Really, the current string is the max length.